packages/language-support/src/dbSchema.ts

```diff
@@ -9,6 +9,7 @@ export interface DbSchema {
   roleNames?: string[];
   parameters?: Record<string, unknown>;
   propertyKeys?: string[];
-  procedures?: Record<string, Neo4jProcedure>;
-  functions?: Record<string, Neo4jFunction>;
+  procedures?: Partial<Record<CypherVersion, Record<string, Neo4jProcedure>>>;
+  functions?: Partial<Record<CypherVersion, Record<string, Neo4jFunction>>>;
```

Comment:
We need a different procedure registry for each version because the same function / procedure could have a different signature in different versions.

At the moment there are not many examples of methods that have different signatures (just `apoc.cypher.runTimeboxed`), but there will be in the future.